zipkin/include/zipkin/tracer.h

```diff
@@ -123,6 +158,39 @@ class Tracer : public TracerInterface {
    */
   void setReporter(ReporterPtr reporter);
```

Comment:
Weren't you going to remove the setReporter method?

Reply:
Fixed.